### ISSUE TITLE
main.collection shows error while double clicked

### ISSUE THREAD

**Issue by JunedQureshi** [defold/defold#5123]
### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2020-08-20T10:47:48.828507</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>5797e374b07cac16d71a29c264bd89e63cfe7652</td></tr>
<tr><td>Defold engine sha</td><td>dedf1ed10d96c92df6e361f5494531c79af4c1cf</td></tr>
<tr><td>Defold version</td><td>1.2.172</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"6dc73b19b61440fe85e203976c183245"'>6dc73b19b61440fe85e203976c183245</a></td></tr>
<tr><td>GPU</td><td>Mesa Intel(R) HD Graphics 520 (SKL GT2)</td></tr>
<tr><td>GPU Driver</td><td>OpenGL ES-CM 1.1 Mesa 20.0.8</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Linux</td></tr>
<tr><td>OS version</td><td>5.4.0-42-generic</td></tr>
<table>

**Comment by britzl** [comment]
Could you please try this: https://defold.com/faq/faq/#q-why-does-the-defold-editor-crash-when-i-try-to-create-a-new-project-or-open-an-existing-one

**Comment by JunedQureshi** [comment]
Defold engine runs smoothly. I can Build and run the code. But when I try to open main.collection it throws an error. 

java.lang.ClassCastException: class jogamp.opengl.es1.GLES1Impl cannot be cast to class com.jogamp.opengl.GL2ES2 (jogamp.opengl.es1.GLES1Impl and com.jogamp.opengl.GL2ES2 are in unnamed module of loader 'app')

After clicking Dismiss I started getting one more error.

java.lang.ClassCastException: Unknown

After clicking Dismiss I get this error.

java.lang.ClassCastException: class jogamp.opengl.es1.GLES1Impl cannot be cast to class com.jogamp.opengl.GL2ES2 (jogamp.opengl.es1.GLES1Impl and com.jogamp.opengl.GL2ES2 are in unnamed module of loader 'app')

Clicking Dismiss

java.lang.ClassCastException: Unknown

I tried running the following with no success. 
$ export MESA_GL_VERSION_OVERRIDE=2.1
$ export MESA_GL_VERSION_OVERRIDE=3.1

I installed libopenal1 still no success.
$ apt-get install libopenal-dev

No Success.

**Comment by britzl** [comment]
> I installed libopenal1 still no success.
> $ apt-get install libopenal-dev

This is only for problems when playing sounds.

> I tried running the following with no success.
> $ export MESA_GL_VERSION_OVERRIDE=2.1
> $ export MESA_GL_VERSION_OVERRIDE=3.1

Other users with the same problem have been able to run Defold after exporting that environment variable. Are you launching Defold from the same terminal/console window as you did the "export MESA_GL_VERSION_OVERRIDE=2.1"?

**Comment by JunedQureshi** [comment]
Greetings !
 I apologize for the delay. 
I downloaded a fresh copy of defold engine for Linux using the below link.

https://defold.com/gr/manuals/install/
From a terminal, I located the archive file “Defold-x86_64-linux.zip", unzipped it to a target Directory called “Defold”.

$ unzip Defold-x86_64-linux.zip -d Defold

I tried running the following command at the terminal.

$ export MESA_GL_VERSION_OVERRIDE=2.1

$ cd Defold
$ ./Defold

I successfully launched he defold game engine and I was able to access "main.collection"

The issue is resolved. Thank you very much britzl.
